exercises/solutions/solution_02.py

```diff
@@ -3,10 +3,9 @@
 def test_show_more_button_of_popular_flights_displays_increment_of_currently_shown_popular_flight_cards(page):
     # 1. On the Kiwi.com website get the count of tiles/cards in the Popular flights section
     # and store it into a variable
-    # 1.1. Open the kiwi.com website (https://www.kiwi.com/en/)
+    # 1.1. Open the kiwi.com website (https://www.kiwi.com/en/) and accept cookies
     page.goto("https://www.kiwi.com/en/")
     page.locator("[data-test='CookiesPopup-Accept']").click()
-    assert page.locator("text=Book cheap flights other sites simply can’t find.").is_visible()
```

Comment:
Shouldn't we keep this assertion here, for the script to be more test-like (verifying the state of the application during the test run)?